modelon/impact/client/entities/experiment.py

```diff
@@ -572,3 +597,139 @@ def get_solver_options(self) -> SolverOptions:
         """Return a SolverOptions object."""
         analysis = self._get_info()["experiment"]["base"]["analysis"]
         return SolverOptions(analysis.get("solverOptions", {}), self.custom_function)
+
+    def _get_initialize_from(
+        self, modifiers: Dict[str, Any]
+    ) -> Optional[CaseOrExperimentOrExternalResult]:
+        if "initializeFrom" in modifiers:
+            exp_id = modifiers["initializeFrom"]
+            return Experiment(
+                workspace_id=self._workspace_id, exp_id=exp_id, service=self._sal
```

Comment:
I think you need remote call to retrieve this experiment by ID from server so that even the "info" is included and also implicitly check that the reference is valid.